src/main.rs

```diff
@@ -161,6 +176,12 @@ fn main() {
             }
         }
         ("explain", Some(sub_m)) => print::explain(sub_m.clone()),
-        _ => {}
+        ("completions", Some(sub_m)) => {
+            let shell_name = sub_m.value_of("shell").expect("Shell name missing.");
+            let shell = Shell::from_str(shell_name).expect("Invalid shell");
+
+            app.gen_completions_to("starship", shell, &mut io::stdout().lock());
+        }
+        (command, _) => panic!("Invalid subcommand: {}", command),
```

Comment:
Is there a reason we'd want to add this? It seems it's not used as _clap_ will catch any invalid sub commands.

Reply:
Better correct than sorry, right? It's better to have a panic that'll never happen than have incorrect logic that relies on a check at-a-distance.

Reply:
The obvious protection here is that if someone adds a subcommand but doesn't update this `match`, it'll break loudly instead of silently doing the wrong thing.

Reply:
Makes sense.

Just want to mention that `unreachable!()` might be more expressive, in that we don't actually expect it to be reached.

Reply:
Sure, that makes sense; I'll make the.change

Reply:
Done